### DISCUSSION TITLE
Add Comments To React/Next.js Website

### DISCUSSION THREAD

**Discussion by giscus** [carlson-technologies/coffeeclass.io#50] [Website Comments]
# Add Comments To React/Next.js Website

How to add comments to a Next.js/React website? In this tutorial, we'll use utteranc - a comment solution which uses GitHub issues to store comments.

https://www.coffeeclass.io/articles/add-comments-to-nextjs-website

**Comment by ghost** [comment]
Test of de comment

**Reply by bjcarlson42** [reply]
Works!
